styles/hydrogen.less

```diff
@@ -289,20 +289,19 @@ svg:first-child {
     font-family: monospace;
     &:not(.scroll-list) {
       flex: 1;
-      border: solid 2px red;
```

Comment:
I have a bad habit of using this almost like `console.log` debugging but for css. Sorry about that!

Reply:
That bad habit kick started my open source development career! ;-)